samples/sample_encode/src/sample_encode.cpp

```diff
@@ -1433,9 +1433,9 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
     //    return MFX_ERR_UNSUPPORTED;
     //}
 
-    if ((pParams->nMaxSliceSize) && (pParams->CodecId != MFX_CODEC_AVC))
+    if ((pParams->nMaxSliceSize) && !((pParams->CodecId == MFX_CODEC_AVC) || (pParams->CodecId == MFX_CODEC_HEVC)))
```

Comment:
honestly, I think that
```
if (pParams->nMaxSliceSize && (pParams->CodecId != MFX_CODEC_AVC) && (pParams->CodecId != MFX_CODEC_HEVC))
```
is much better

Reply:
done